.github/workflows/cloud-deploy.yml

```diff
@@ -0,0 +1,105 @@
+name: Release Prod
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - prod
+
+jobs:
+  pre-release:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Start Deployment Notification
+        uses: actions/slack@master
+        with:
+          channel: '#eng-feed-deployments'
+          message: 'Release has started'
+        env:
+          SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+
+  release-api:
+    needs: [pre-release]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: ./.github/workflows/cloud-deploy/cloud-deploy-api.yml
+
+  release-embed:
+    needs: [pre-release]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: ./.github/workflows/cloud-deploy/cloud-deploy-api.yml
+
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Should these `jobs` have different arguments somewhere?‏

Reply:
Your right